lib/statsd/instrument/prometheus/aggregators/sum.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module StatsD
+  module Instrument
+    module Prometheus
+      module Aggregators
+        class Sum < Base
+          def aggregate
+            last_datagram = datagrams.pop
+            sum = datagrams.inject(last_datagram.value) do |accumulator, datagram|
+              accumulator + datagram.value
```

Comment:
For counters: https://github.com/coralogix/statsd-coralogix-backend/blob/master/lib/coralogix.js#L167